model/shared/src/main/scala/explore/model/ObsSummary.scala

```diff
@@ -56,7 +57,8 @@ case class ObsSummaryWithConstraints(
   override val status:       ObsStatus,
   override val activeStatus: ObsActiveStatus,
   override val duration:     Duration,
-  targetEnvId:               TargetEnvironment.Id
+  targetEnvId:               TargetEnvironment.Id,
+  targetIds:                 Set[Target.Id]
```

Comment:
Are these science target ids or may they refer to any target? Why are they needed here?